src/main/resources/lang/fr_fr.json

```diff
@@ -0,0 +1,115 @@
+{
+	"language": "Français",
+
+	"mapping_format.enigma_file": "Fichier Enigma",
+	"mapping_format.enigma_directory": "Répertoire Enigma",
+	"mapping_format.tiny_v2": "Tiny v2",
+	"mapping_format.tiny_file": "Fichier Tiny",
+	"mapping_format.srg_file": "Fichier SRG",
+	"mapping_format.proguard": "Proguard",
+	"type.methods": "Méthodes",
+	"type.fields": "Champs",
+	"type.parameters": "Paramètres",
+	"type.classes": "Classes",
+
+	"menu.file": "Fichier",
+	"menu.file.jar.open": "Ouvrir le jar...",
+	"menu.file.jar.close": "Fermer le jar",
+	"menu.file.mappings.open": "Ouvrir les mappings...",
+	"menu.file.mappings.save": "Enregistrer les mappings",
+	"menu.file.mappings.save_as": "Enregistrer les mappings sous...",
+	"menu.file.mappings.close": "Fermer les mappings",
+	"menu.file.mappings.drop": "Supprimer les mappings invalides",
+	"menu.file.export.source": "Exporter la source...",
+	"menu.file.export.jar": "Exporter le jar...",
+	"menu.file.stats": "Statistiques des mappings...",
+	"menu.file.stats.title": "Choisir les membres inclus",
+	"menu.file.stats.generate": "Générer les statistiques",
+	"menu.file.exit": "Quitter",
+	"menu.view": "Affichage",
+	"menu.view.themes": "Thèmes",
+	"menu.view.themes.default": "Par défaut",
+	"menu.view.themes.darcula": "Darcula",
+	"menu.view.themes.system": "Système",
+	"menu.view.themes.none": "Aucun (JVM par défaut)",
+	"menu.view.languages": "Langues",
+	"menu.view.languages.title": "Modifier la langue",
+	"menu.view.languages.summary": "Le nouveau langage sera appliqué lors du prochain redémarrage.",
```

Comment:
```suggestion
	"menu.view.languages.summary": "La nouvelle langue sera appliqué lors du prochain redémarrage.",
```

Reply:
Is this French French or Quebec French?

Reply:
Both. "langage" refers to all languages in general (like the word "speech" in English), while "langue" is used for a specific language. By the way, "langue" is correctly used instead of "language" in the sentence right above.

Reply:
Ah, just making sure there isn't cases like `Fixing "color" typo with "colour"`

Reply:
Translation is French French. "nouvelle langue" sounds a bit weird, but it's the true translation, so I change it

Reply:
you forgot to update "appliqué**e**"